src/ferc_eia_match/__init__.py

```diff
@@ -24,5 +24,8 @@
 __downloadurl__ = "https://github.com/catalyst-cooperative/ferc_eia_match"
 
 # Create a root logger for use anywhere within the package.
-logger = logging.getLogger(__name__)
+logging.basicConfig(level="INFO")
+logger = logging.getLogger("ferc_eia_match")
+log_format = "%(asctime)s [%(levelname)8s] %(name)s:%(lineno)s %(message)s"
```

Comment:
This is totally fine for this work-in-progress repo, but we'll need to remember to update it if/when we pull this code into PUDL so we aren't subverting logging flexibility project-wide.

Reply:
Got it. So we'd switch to using the PUDL logger? Does it make sense to start using the PUDL logger now?